From: Kai Uwe Broulik <kde@privat.broulik.de>
Date: Sat, 11 Feb 2017 17:06:15 +0100
Subject: [UDisks 2 Device] Add description for Floppy Disk

Ensures a floppy disk shows up as "Floppy Disk" instead of "0 Byte Removable Media"

CHANGELOG: Floppy disks now show up as "Floppy Disk" instead of "0 B Removable Media"

Differential Revision: https://phabricator.kde.org/D4570
---
 src/solid/devices/backends/udisks2/udisksdevice.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/solid/devices/backends/udisks2/udisksdevice.cpp b/src/solid/devices/backends/udisks2/udisksdevice.cpp
index 5208ece..b0a6b51 100644
--- a/src/solid/devices/backends/udisks2/udisksdevice.cpp
+++ b/src/solid/devices/backends/udisks2/udisksdevice.cpp
@@ -573,6 +573,8 @@ QString Device::volumeDescription() const
                 description = tr("Hard Drive");
             }
         }
+    } else if (drive_type == Solid::StorageDrive::Floppy) {
+        description = tr("Floppy Disk");
     } else {
         if (drive_is_removable) {
             description = tr("%1 Removable Media", "%1 is the size").arg(size_str);
